samples/tfm_integration/psa_protected_storage/README.rst

```diff
@@ -0,0 +1,54 @@
+.. psa_protected_storage:
+
+PSA Protected Storage
+#####################
+
+This sample demonstrates how the Protected Storage (PS) API can be used for storing data.
```

Comment:
I would suggest providing some more context here on what PS is (since there is also ITS), such as:

```suggestion
This sample demonstrates how the Protected Storage (PS) API can be used for storing data.

Protected storage data is (by default) encrypted, with optional authentication and rollback
protection.

The default crypto algorithm is ``AES-128-GCM``. The encryption key
is derived from the Hardware Unique Key (HUK), which is often set via
device fuses, etc.

TF-M includes a maximum number of PS records, set via ``PS_NUM_ASSETS``
(default 10 as of TF-M 1.3), and a maximum record size, set via
``PS_MAX_ASSET_SIZE`` (default of 2048 as of TF-M 1.3.0). These defaults may be
different depending on the platform being used.
```

Reply:
This is great, thanks!